SimplenoteUITests/SimplenoteUISmokeTestsSettings.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+class SimplenoteUISmokeTestsSettings: XCTestCase {
+
+    override class func setUp() {
+        app.launch()
+        getToAllNotes()
+        _ = attemptLogOut()
+        EmailLogin.open()
+        EmailLogin.logIn(email: testDataExistingEmail, password: testDataExistingPassword)
+        NoteList.waitForLoad()
+    }
+
+    override func setUpWithError() throws {
+        getToAllNotes()
+        NoteList.trashAllNotes()
+        Trash.empty()
+        NoteList.openAllNotes()
+    }
+
+    override class func tearDown() {
+        Settings.open()
+        Settings.condensedModeDisable()
+        Settings.close()
+    }
```

Comment:
Disabling the condensed mode at the end of the suite, because tests in other areas (like Search) will fail under condensed mode.